.github/workflows/coverage.yml

```
@@ -75,3 +78,10 @@ jobs:
         uses: coverallsapp/github-action@master
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Compress coverage report
+        run: xz -T0 -9e -c coverage/lcov.info > coverage/lcov.info.xz
+      - name: Upload coverage artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: coverage-${{ replace(env.BRANCH_NAME, '/', '-') }}-${{ env.COMMIT_SHA }}
```

actionlint (1.7.7)

undefined function "replace". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)